[Build] Exclude specific module (r.geomorphon) at compilation

## Describe the bug
I can't finish the GRASS build due to one module not compiling:

`Cannot parse interface for module r.geomorphon. Empty strings will be placed instead of description and keywords. Reason: Unable to fetch interface description for command '<r.geomorphon>'.`

This is certainly my fault. Is there any way to disable one specific module at configuration time?

## To reproduce
```
    ./configure \
    --prefix=/opt/$pkgname \
    --with-freetype-includes=/usr/include/freetype2 \
    --with-wxwidgets \
    --with-readline \
    --with-pthread \
    --with-netcdf \
    --with-nls \
    --with-geos \
    --with-postgres \
    --with-pdal \
    --with-bzlib \
    --with-zstd

    --disable-geomorphon ??

  make
```

## System description
- ArchLinux
- git master

## Additional context
<!-- Add any other context about the problem. -->

Is LC_ALL=C necessary?
```
../include/Make/Rules.make:16: Warnung: Das Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu“ wird überschrieben
../include/Make/Rules.make:16: Warnung: Das alte Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu“ wird ignoriert
../include/Make/Rules.make:16: Warnung: Das Rezept für das Ziel „./src/grass/bin.x86_64-pc-linux-gnu“ wird überschrieben
../include/Make/Rules.make:16: Warnung: Das alte Rezept für das Ziel „./src/grass/bin.x86_64-pc-linux-gnu“ wird ignoriert
../include/Make/Rules.make:16: Warnung: Das Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/include/grass“ wird überschrieben
../include/Make/Rules.make:16: Warnung: Das alte Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/include/grass“ wird ignoriert
../include/Make/Rules.make:16: Warnung: Das Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/lib“ wird überschrieben
../include/Make/Rules.make:16: Warnung: Das alte Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/lib“ wird ignoriert
../include/Make/Rules.make:16: Warnung: Das Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/bin“ wird überschrieben
../include/Make/Rules.make:16: Warnung: Das alte Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/bin“ wird ignoriert
../include/Make/Rules.make:16: Warnung: Das Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/etc“ wird überschrieben
../include/Make/Rules.make:16: Warnung: Das alte Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/etc“ wird ignoriert
../include/Make/Rules.make:16: Warnung: Das Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/driver“ wird überschrieben
../include/Make/Rules.make:16: Warnung: Das alte Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/driver“ wird ignoriert
../include/Make/Rules.make:16: Warnung: Das Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/driver/db“ wird überschrieben
../include/Make/Rules.make:16: Warnung: Das alte Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/driver/db“ wird ignoriert
../include/Make/Rules.make:16: Warnung: Das Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/fonts“ wird überschrieben
../include/Make/Rules.make:16: Warnung: Das alte Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/fonts“ wird ignoriert
../include/Make/Rules.make:16: Warnung: Das Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/docs“ wird überschrieben
../include/Make/Rules.make:16: Warnung: Das alte Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/docs“ wird ignoriert
../include/Make/Rules.make:16: Warnung: Das Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/docs/html“ wird überschrieben
../include/Make/Rules.make:16: Warnung: Das alte Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/docs/html“ wird ignoriert
../include/Make/Rules.make:16: Warnung: Das Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/docs/man“ wird überschrieben
../include/Make/Rules.make:16: Warnung: Das alte Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/docs/man“ wird ignoriert
../include/Make/Rules.make:16: Warnung: Das Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/docs/man/man1“ wird überschrieben
../include/Make/Rules.make:16: Warnung: Das alte Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/docs/man/man1“ wird ignoriert
../include/Make/Rules.make:16: Warnung: Das Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/utils“ wird überschrieben
../include/Make/Rules.make:16: Warnung: Das alte Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/utils“ wird ignoriert
../include/Make/Rules.make:19: Warnung: Das Rezept für das Ziel „OBJ.x86_64-pc-linux-gnu“ wird überschrieben
../include/Make/Rules.make:19: Warnung: Das alte Rezept für das Ziel „OBJ.x86_64-pc-linux-gnu“ wird ignoriert
../include/Make/Rules.make:56: Warnung: Das Rezept für das Ziel „clean“ wird überschrieben
../include/Make/Rules.make:56: Warnung: Das alte Rezept für das Ziel „clean“ wird ignoriert
WARNING: Weisberg-Bingham's W'' cannot be used for n < 50 or n > 99
make[4]: *** [../../include/Make/Html.make:14: r.geomorphon.tmp.html] Fehler 1
Makefile:13: Warnung: Das Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/etc/d.polar“ wird überschrieben
../../include/Make/ScriptRules.make:19: Warnung: Das alte Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/etc/d.polar“ wird ignoriert
Makefile:16: Warnung: Das Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/etc/db.test“ wird überschrieben
../../include/Make/ScriptRules.make:19: Warnung: Das alte Rezept für das Ziel „./src/grass/dist.x86_64-pc-linux-gnu/etc/db.test“ wird ignoriert
Creating translations (= 'make mo')
3361 übersetzte Meldungen, 164 ungenaue Übersetzungen, 747 unübersetzte Meldungen.
0 übersetzte Meldungen, 4272 unübersetzte Meldungen.
646 übersetzte Meldungen, 86 ungenaue Übersetzungen, 3540 unübersetzte Meldungen.
2192 übersetzte Meldungen, 142 ungenaue Übersetzungen, 1938 unübersetzte Meldungen.
po/grasswxpy_de.po:20658: Mehrfachdefinition der Meldung...
po/grasswxpy_de.po:4581: ...dies ist die Stelle der ersten Definition
msgfmt: found 1 fatal error
make[3]: *** [Makefile:100: ./src/grass/dist.x86_64-pc-linux-gnu/locale/de/LC_MESSAGES/grasswxpy.mo] Fehler 1
```

`[./r.geomorphon]$ LC_ALL=C make`

```
if [ "./grass/dist.x86_64-pc-linux-gnu/bin/r.geomorphon" != "" ] ; then GISRC=./grass/dist.x86_64-pc-linux-gnu/demolocation/.grassrc84 GISBASE=./grass/dist.x86_64-pc-linux-gnu PATH="./grass/dist.x86_64-pc-linux-gnu/bin:./grass/dist.x86_64-pc-linux-gnu/bin:./grass/dist.x86_64-pc-linux-gnu/scripts:$PATH" PYTHONPATH="./grass/dist.x86_64-pc-linux-gnu/etc/python:./grass/dist.x86_64-pc-linux-gnu/gui/wxpython:$PYTHONPATH" LD_LIBRARY_PATH="./grass/dist.x86_64-pc-linux-gnu/bin:./grass/dist.x86_64-pc-linux-gnu/bin:./grass/dist.x86_64-pc-linux-gnu/scripts:./grass/dist.x86_64-pc-linux-gnu/lib:./grass/dist.x86_64-pc-linux-gnu/lib:/usr/lib/dri/:" LC_ALL=C LANG=C LANGUAGE=C ./grass/dist.x86_64-pc-linux-gnu/bin/r.geomorphon --html-description < /dev/null | grep -v '</body>\|</html>\|</div> <!-- end container -->' > r.geomorphon.tmp.html ; fi
make: *** [../../include/Make/Html.make:14: r.geomorphon.tmp.html] Error 1
rm r.geomorphon.tmp.html
```

--- a/locale/po/grasswxpy_de.po
+++ b/locale/po/grasswxpy_de.po
@@ -6948,8 +6948,8 @@ msgstr "SIMWE Oberflächenabflussmodellierung"
 #: ../gui/wxpython/menustrings.py:428 ../gui/wxpython/menustrings.py:1391
 msgid "Overland flow hydrologic simulation using path sampling method (SIMWE)."
 msgstr ""
-"Hydrologische Simulation des Oberflächenabfluss mittels \"path sampling"
-"\" (SIMWE)."
+"Hydrologische Simulation des Oberflächenabfluss mittels \"path "
+"sampling\" (SIMWE)."
 
 #: ../gui/wxpython/menustrings.py:429 ../gui/wxpython/menustrings.py:1392
 msgid "SIMWE Sediment flux modeling"
@@ -6960,8 +6960,8 @@ msgid ""
 "Sediment transport and erosion/deposition simulation using path sampling "
 "method (SIMWE)."
 msgstr ""
-"Sedimenttransport und Erosions-/Deposition-Simulation mittels \"path sampling"
-"\" (SIMWE)."
+"Sedimenttransport und Erosions-/Deposition-Simulation mittels \"path "
+"sampling\" (SIMWE)."
 
 #: ../gui/wxpython/menustrings.py:431 ../gui/wxpython/menustrings.py:1394
 msgid "Topographic index map"
@@ -13705,13 +13705,13 @@ msgstr ""
 #: ../gui/wxpython/image2target/ii2t_gis_set.py:811
 #, python-format
 msgid ""
-"Do you want to continue with deleting mapset <%(mapset)s> from location <"
-"%(location)s>?\n"
+"Do you want to continue with deleting mapset <%(mapset)s> from location "
+"<%(location)s>?\n"
 "\n"
 "ALL MAPS included in this mapset will be PERMANENTLY DELETED!"
 msgstr ""
-"Wollen Sie mit dem Löschen des Mapsets <%(mapset)s> der Location <"
-"%(location)s> fortfahren?\n"
+"Wollen Sie mit dem Löschen des Mapsets <%(mapset)s> der Location "
+"<%(location)s> fortfahren?\n"
 "\n"
 "ALLE KARTEN in diesem Mapset werden UNWIDERRUFLICH GELÖSCHT!"
 
@@ -15989,8 +15989,8 @@ msgstr ""
 #, python-format
 msgid ""
 "Scatter plot cannot be added.\n"
-"Multiple of bands ranges <%s:%d * %s:%d = %d> is higher than maximum limit <"
-"%d>.\n"
+"Multiple of bands ranges <%s:%d * %s:%d = %d> is higher than maximum limit "
+"<%d>.\n"
 msgstr ""
 
 #: ../gui/wxpython/iscatt/controllers.py:351
@@ -18133,8 +18133,8 @@ msgstr ""
 #: ../gui/wxpython/dbmgr/base.py:2629
 #, python-format
 msgid ""
-"Unable to rename column. Column <%(column)s> doesn't exist in the table <"
-"%(table)s>."
+"Unable to rename column. Column <%(column)s> doesn't exist in the table "
+"<%(table)s>."
 msgstr ""
 "Kann die Spalte nicht umbenennen. Die Spalte <%(column)s> existiert nicht in "
 "der Tabelle <%(table)s>."
@@ -20652,17 +20652,6 @@ msgstr "Beschriftungen:"
 
 #~ msgid "GRASS GIS Raster Map Calculator"
 #~ msgstr "GRASS GIS Rasterkartenrechner"
-
-#: ../gui/wxpython/gmodeler/dialogs.py:318
-#, fuzzy, python-format
-msgid ""
-"'%s' is not a GRASS tool.\n"
-"\n"
-"Unable to add new action to the model."
-msgstr ""
-"'%s' ist kein GRASS Werkzeug.\n"
-"\n"
-"Kann keine neue Aktion zum Modell hinzufügen."
 
 #~ msgid "GRASS GIS 3D Raster Map Calculator"
 #~ msgstr "GRASS GIS 3D Raster Map Calculator"

